sdk/src/integrationTest/java/com/meniga/sdk/models/upcoming/MenigaUpcomingIntegrationTest.java

```diff
@@ -42,6 +38,7 @@ public void setUp() {
 	}
 
 	@Test
+	@Ignore("Not in the API anymore")
```

Comment:
What do you think about removing this test instead of ignoring?

Reply:
I would create a separate PR with removing this upcoming code in favor of the new one.